package/yast2-auth-client.changes

```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
```

Comment:
OK look at this hehe..
http://yastgithubio.readthedocs.org/en/latest/contributing/

Anyways - changelog for 3.2 is committed below.